devito/operator/operator.py

```diff
@@ -305,26 +305,8 @@ def _lower_exprs(cls, expressions, **kwargs):
             else:
                 dimension_map = {}
 
-            mapper = {}
-
             # Handle Functions (typical case)
-            for f in retrieve_functions(expr):
```

Comment:
Note: These lines were literally copy-pasted from `basic` and bypassing `indexify` in most cases, so fixed indexify and removed.